apps/render_cf.py

```diff
@@ -59,9 +270,33 @@ def render_costs(job_types: dict, cost_profile: str) -> None:
         json.dump(costs, f, indent=2)
 
 
+def validate_job_spec(job_type: str, job_spec: dict) -> None:
+    # Non-comprehensive job spec validator. More checks could be added if we want to be more thorough.
+
```

Comment:
Does this cover everything that can silently fail? If so, I think its fine.
```suggestion
```

Reply:
I guess I was trying to leave open the possibility that this function becomes like a definitive reference for our job spec syntax. But I'm fine with deleting the comment if you think it just causes confusion.